common/types/pb/pb.go

```diff
@@ -121,7 +121,12 @@ func CollectFileDescriptorSet(message proto.Message) (*descpb.FileDescriptorSet,
 	fdMap := map[string]*descpb.FileDescriptorProto{}
 	fd, _ := descriptor.ForMessage(message.(descriptor.Message))
 	fdMap[fd.GetName()] = fd
-	deps := fd.GetDependency()
+	// Initialize list of dependencies
```

Comment:
I don't think this is doing anything interesting. Were there other changes in other files?

Reply:
I see, you want a copy? In theory, the `FileDescriptorProto` is immutable, but if not, then this is the simpler way to specify what's being done here:

`deps := fd.GetDependency()[:]`